docs/en/ros2/px4_ros2_control_interface.md

```diff
@@ -400,6 +401,134 @@ _goto_setpoint->update(
   max_heading_rate_rad_s);
 ```
 
+#### Fixed-Wing Lateral and Longitudinal Setpoint (FwLateralLongitudinalSetpointType)
+
+<Badge type="warning" text="Fixed wing only" />
+
+::: info
+This setpoint type is only supported for fixed-wing vehicles.
+:::
+
+Use the [`px4_ros2::FwLateralLongitudinalSetpointType`](https://auterion.github.io/px4-ros2-interface-lib/classpx4__ros2_1_1FwLateralLongitudinalSetpointType.html) to directly control the lateral and longitudinal dynamics of a fixed-wing vehicle — that is, side-to-side motion (turning/banking) and forward/upward motion (speeding up/climbing) respectively.
+This setpoint is streamed to the PX4 _FwLateralLongitudinalControl_ module, which decouples lateral and longitudinal inputs while ensuring that vehicle limits are respected.
+
+To control the vehicle, at least one lateral **and** one longitudinal setpoint must be provided:
+
+1. Of the longitudinal inputs: either `altitude` or `height_rate` must be finite to control vertical motion.
+   If both are set to `NAN`, the vehicle will maintain its current altitude.
+2. Of the lateral inputs: at least one of `course`, `airspeed_direction`, or `lateral_acceleration` must be finite.
+
+For a detailed description of the controllable parameters, please refer to message definitions ([FixedWingLateralSetpoint](../msg_docs/FixedWingLateralSetpoint.md) and [FixedWingLongitudinalSetpoint](../msg_docs/FixedWingLongitudinalSetpoint.md)).
+
+##### Basic Usage
+
+This setpoint type offers multiple update methods, each allowing you to specify an increasing number of setpoints. In the simplest case, you can provide a `course` and an `altitude` setpoint:
```

Comment:
YOu'll note I use line breaks on sentences. This is effectively cosmetic. I prefer it though because for smaller changes the github change control makes it easier to what is happening.

```suggestion
This setpoint type offers multiple update methods, each allowing you to specify an increasing number of setpoints.
In the simplest case, you can provide a `course` and an `altitude` setpoint:
```